infrastructure/code/backend_service.go

```diff
@@ -174,10 +174,13 @@ func (s *SnykCodeHTTPClient) doCall(ctx context.Context,
 
 		response, body, err := s.httpCall(req)
 		responseBody = body
-		log.Trace().Str("response.Status", response.Status).
-			Str("responseBody", string(responseBody)).
-			Str("snyk-request-id", requestId).
-			Msg("RECEIVED FROM REMOTE")
+
+		if response != nil && responseBody != nil {
+			log.Trace().Str("response.Status", response.Status).
```

Comment:
Would probably make sense to log `RECEIVED FROM REMOTE` in that case anyway as it'd help with debugging.

Reply:
Done!